### PR TITLE
theme Flatten added

### PR DESCRIPTION
**Pull request by qwersteve07**
Pls let me know if anything I should fix or changed, thx a lot :)

### CODE REVIEW

File: SimpleSpotify/color.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cover_overlay_and_shadow              = 222
+
+pressing_fg                           = 999999
+scrollbar_fg_and_selected_row_bg      = yellow
+pressing_button_fg                    = green
+pressing_button_bg                    = 111
+selected_button                       = brown
+miscellaneous_bg                      = 111
+miscellaneous_hover_bg                = purple
+preserve_1                            = pink
```
Comment (morpheusthewhite) [inline SimpleSpotify/color.ini:20]: You probably left this by mistake, as well as the empty README in the same folder

File: SimpleSpotify/user.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+.button.button-green {
+  color: #ffffff;
+}
+
+.glue-page-header__content .glue-page-header__title {
+  color: #ffffff;
+}
+
+/*# sourceMappingURL=user.css.map */
```
Comment (morpheusthewhite) [inline SimpleSpotify/user.css:119]: You probably left this by mistake

File: Flatten/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Flatten - spotify-based flat design
```
Comment (morpheusthewhite) [inline Flatten/README.md:1]: I'd prefer to have as title only the theme name in order to be consistent

**Comment by qwersteve07** [comment]
fixed all problems!

**Pull request merged**